### PR TITLE
Move StyleSheetLoader to correct package.

### PR DESCRIPTION
**Pull request by tinydylan**
Related Ticket:

Description of Changes:
* Move StyleSheetLoader to correct package.

Pre-checks:
* [X] Changelog entry added
* [X] Tests have been added (if applicable)
* [X] Branch prefixed with `feature/` for new features (if applicable)
* [X] License headers added on new files (if applicable)

Review:
* [X] Milestone set
* [x] Review comments resolved

GitHub issues (if applicable):

### CODE REVIEW

**Comment by tinydylan** [comment]
@lnewson Sorry about the last PR. Is this better?

File: modules/tinymce/src/core/main/ts/api/Tinymce.ts [hunk 1]
```diff
@@ -99,6 +99,7 @@ export interface TinyMCE extends EditorManager {
     ControlSelection: (selection: Selection, editor: Editor) => ControlSelection;
     BookmarkManager: (selection: Selection) => BookmarkManager;
     Selection: (dom: DOMUtils, win: Window, serializer, editor: Editor) => Selection;
+    StyleSheetLoader: StyleSheetLoader;
```
Comment (lnewson) [inline modules/tinymce/src/core/main/ts/api/Tinymce.ts:102]: This only exposes the types, not the actual constructor. So it likely needs to be added to the actual exported JS as well (eg ~line 220).

Comment (tinydylan) [reply modules/tinymce/src/core/main/ts/api/Tinymce.ts:102]: I'll get it right one day. 
How about now?

Comment (lnewson) [reply modules/tinymce/src/core/main/ts/api/Tinymce.ts:102]: Yep, looks good to me now, thanks @tinydylan! :+1:

Comment (tinydylan) [reply modules/tinymce/src/core/main/ts/api/Tinymce.ts:102]: Cheers, mate. Thanks for you help.

**Comment by lnewson** [comment]
Just remembered one other thing sorry @tinydylan. For new globals, you also need to add it to `modules/tinymce/src/core/main/json/globals.json` otherwise you won't be able to use it elsewhere in plugins/themes.

We really need to document this, so I'll take a look at adding something to `CONTRIBUTING.md`.

**Comment by tinydylan** [comment]
> Just remembered one other thing sorry @tinydylan. For new globals, you also need to add it to `modules/tinymce/src/core/main/json/globals.json` otherwise you won't be able to use it elsewhere in plugins/themes.
> 
> We really need to document this, so I'll take a look at adding something to `CONTRIBUTING.md`.

I've added to `globals.json` and `CONTRIBUTING.md`. I'm not quite sure of the wording in `CONTRIBUTING.md`, so feedback welcome.

**Comment by lnewson** [comment]
> I've added to `globals.json` and `CONTRIBUTING.md`. I'm not quite sure of the wording in `CONTRIBUTING.md`, so feedback welcome.

Looks good to me, thanks heap for doing that for me! :trophy:

**Pull request merged**